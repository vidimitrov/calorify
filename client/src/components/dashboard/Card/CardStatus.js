import styled from 'styled-components';

const CardStatus = styled.div`
  background-color: ${props => (props.inRange ? 'rgb(153, 204, 0)' : '#ff387b')};
  height: 100%;
  width: 10px;
  border-radius: 3px;
`;

export default CardStatus;
