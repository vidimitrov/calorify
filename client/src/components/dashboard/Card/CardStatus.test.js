import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CardStatus from './CardStatus';

it('Dashboard/Card/CardStatus component renders correctly', () => {
  const tree = renderer.create(<CardStatus inRange />).toJSON();
  expect(tree).toMatchSnapshot();
});
