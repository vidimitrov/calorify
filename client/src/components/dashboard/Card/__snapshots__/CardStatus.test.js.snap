// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/Card/CardStatus component renders correctly 1`] = `
.c0 {
  background-color: rgb(153,204,0);
  height: 100%;
  width: 10px;
  border-radius: 3px;
}

<div
  className="c0"
/>
`;
